package.json

```diff
@@ -136,5 +136,6 @@
       ],
       "@semantic-release/github"
     ]
-  }
+  },
+  "dependencies": {}
```

Comment:
Just curious: Is this change required?

Reply:
No, but npm added this for me, so I assume it's good practice